actions_root.go

```diff
@@ -97,6 +97,12 @@ func Fatalf(msg string, args ...interface{}) {
 	defaultAction.Fatalf(msg, args...)
 }
 
+// Infof prints a info-level message. The arguments follow the standard Printf
```

Comment:
I can edit this to say "an info-level" message here and in `actions.go`, if desired.